website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -39,12 +39,13 @@ Go to the workspace's notifications settings and press the "Add Notification" bu
 
 Fill in the following fields to configure your notifications:
 
-* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
+* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
```

Comment:
Also --> Is that the right bookmark? It doesn't seem to be formatted with the traditional #, which is why I'm asking!  I think it could make sense to change it to notification payloads, as I have below. Let me know if you disagree!

```suggestion
* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Refer to [Notification Payloads][#notification-payloads] for details.
```
```suggestion
* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
```